specification/security/resource-manager/Microsoft.Security/preview/2023-08-01-preview/pricings.json

```diff
@@ -0,0 +1,461 @@
+            "$ref": "./examples/Pricings/ListPricingsWithPlanFilter_example.json"
+          },
+          "Get pricings on subscription with resource level filter": {
+            "$ref": "./examples/Pricings/ListPricingsWithResourceFilter_example.json"
+          },
+          "Get pricings on resource": {
+            "$ref": "./examples/Pricings/ListResourcePricings_example.json"
+          }
+        },
+        "tags": [
+          "Pricings"
+        ],
+        "description": "Lists Microsoft Defender for Cloud pricing configurations (both subscription and resource level) of the subscription.",
+        "operationId": "Pricings_List",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ScopeId"
+          },
+          {
+            "$ref": "#/parameters/PlansFilter"
+          },
+          {
+            "$ref": "#/parameters/ResourceLevelFilter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/PricingList"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "PricingList": {
+      "description": "List of pricing configurations response.",
+      "required": [
+        "value"
+      ],
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of pricing configurations",
+          "items": {
+            "$ref": "#/definitions/Pricing"
+          }
+        }
+      }
+    },
+    "Pricing": {
+      "type": "object",
+      "description": "Microsoft Defender for Cloud is provided in two pricing tiers: free and standard. The standard tier offers advanced security capabilities, while the free tier offers basic security features.",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Pricing data",
+          "$ref": "#/definitions/PricingProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "PricingProperties": {
+      "type": "object",
+      "description": "Pricing properties for the relevant scope",
+      "properties": {
+        "pricingTier": {
+          "type": "string",
+          "description": "The pricing tier value. Microsoft Defender for Cloud is provided in two pricing tiers: free and standard. The standard tier offers advanced security capabilities, while the free tier offers basic security features. This field will be deprecated.",
```

Comment:
If we are deprecating this field, let's consider requiring the 'isEnabled' field at the Put request, instead of the pricingTier field?

Reply:
Let's also describe that StandardTier & IsEnabled properties describing the same and will always be in sync.